subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -13,23 +13,25 @@
 // limitations under the License.
 
 [[publishing_maven]]
-== Maven Publishing (new)
+== The Maven Publish Plugin
 
 
 [NOTE]
 ====
-This chapter describes the new <<feature_lifecycle,incubating>> Maven publishing support provided by the Maven Publish Plugin. Eventually this new publishing support will replace publishing via the `Upload` task.
+This chapter describes the new Maven publishing support provided by the Maven Publish Plugin. This new publishing support is the preferred option for publishing artifacts and will eventually replace publishing via the `Upload` task.
 
-If you are looking for documentation on the original Maven publishing support using the `Upload` task please see <<artifact_management>>.
+If you are looking for documentation on the original Maven publishing support using the `Upload` task please see <<artifact_management>> and <<maven_plugin>>.
```

Comment:
I don't like these styles of link because the generated text for them is very wordy. The problem is exacerbated by having two links in a row like this. I would go with:
 
    ... please see the chapters on <<artifact_management,publishing artifacts>> and the <<maven_plugin,old Maven Plugin>>.

Reply:
If you want to just use the title of a chapter, use `_<<[link],[chapter title]>>_`, i.e. make the chapter title italic.

Reply:
Actually, `<<chapter>>` does exactly that, doesn't it? 🤔

Reply:
@marcphilipp Sorry, you're right. My issue is with section links, not chapter ones. I hadn't noticed the difference until you mentioned it.